packages/next/build/babel/plugins/next-page-config.ts

```diff
@@ -110,6 +110,19 @@ export default function nextPageConfig({
           t.returnStatement(t.stringLiteral(inlineGipIdentifier)),
         ])
       },
+      // handles class { static async getInitialProps() {} }
+      ClassMethod(path, state: ConfigState) {
```

Comment:
@ijjk could you have a look at this one

Reply:
This seems correct to me to handle the non-transpiled class methods